notes/0.13.10.markdown

```diff
@@ -128,6 +130,7 @@
 - Fixes launcher configuration to add `sbt-ivy-snapshots` repository to resolve nightly builds. [@eed3si9n][@eed3si9n]
 - Fixes performance issues during tree traversal in the incremental compiler. [#2343][2343] by [@adriaanm][@adriaanm]
 - Fixes the tracking of self types and F-bounded existential types in the incremental compiler. [#2343][2343] by [@adriaanm][@adriaanm]
+- Fixes autoImports for AutoPlugins for global configuration files. [#2399][2399] by [@timcharper][@timcharper]
```

Comment:
Can you mention https://github.com/sbt/sbt/issues/2120 as well. And mention it in the commit ("Fixes #2120") so it auto-closes the ticket.